Sources/KituraWebSocket/WebSocketService.swift

```diff
@@ -49,4 +49,12 @@ public protocol WebSocketService: class {
     /// - Parameter client: The `WebSocketConnection` object that represents the connection over which
     ///                     the client sent the message to this `WebSocketService`
     func received(message: String, from: WebSocketConnection)
+    
+    /// The time in seconds after which, if a connection is has sent no messages, the WebSocket server will send a ping to the connection. If a pong is not received in response, the connection will be closed. If connectionTimeout is nil, no connection cleanup will take place.
+    var connectionTimeout: Int? { get }
+}
+extension WebSocketService {
```

Comment:
Newline above this please.

Reply:
Does this need to be marked `public` or is it implicitly public?

Reply:
It is implicitly public but i can declare it public if that makes the code clearer